src/debug/logging/colors.rs

```diff
@@ -1,4 +1,4 @@
-#[cfg(any(target_os = "windows", feature = "plain"))]
+#[cfg(any(feature = "plain"))]
```

Comment:
```suggestion
#[cfg(feature = "plain")]
```

Reply:
Ops, completely forgot to remove the `any`